recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -17,12 +16,14 @@ class RedisPlusPlusConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "with_tls": [True, False]
+        "with_tls": [True, False],
+        "build_async": [True, False]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_tls": False
+        "with_tls": False,
+        "build_async": False
```

Comment:
what does mean build_async option? Build this project in parallel, or is something related to the execution?

Reply:
> what does mean build_async option? Build this project in parallel, or is something related to the execution?

It makes with libuv support so it can be async

Reply:
Okay, thank for the explanation!